velociraptor-plot

```diff
@@ -137,6 +137,8 @@ if __name__ == "__main__":
         style.use(args.stylesheet)
 
     print_if_debug(f"Generating initial AutoPlotter instance for {args.config}.")
+    print(args.config)
+    quit()
```

Comment:
Should that be kept?

Reply:
no, good find. because the login nodes`/tmp` was overloaded when I was testing this it wasn't installing properly so I didn't catch code-breaking debug stuff I'd left here